apps/admin-gui/src/app/shared/components/dialogs/add-member-to-resource-dialog/add-member-to-resource-dialog.component.html

```diff
@@ -9,37 +9,24 @@ <h1 mat-dialog-title>{{'DIALOGS.ADD_MEMBER_TO_RESOURCE.TITLE' | translate}}</h1>
         >
         <mat-spinner *ngIf="loading" class="mr-auto ml-auto"></mat-spinner>
         <div *ngIf="!loading">
-          <div>
-            <mat-form-field class="input-style">
-              <input
-                [formControl]="facilityCtrl"
-                [matAutocomplete]="autoFacility"
-                aria-label="Number"
-                matInput
-                placeholder="{{ 'DIALOGS.ADD_MEMBER_TO_RESOURCE.FILTER' | translate}}"
-                type="text" />
-              <mat-autocomplete #autoFacility="matAutocomplete" autoActiveFirstOption>
-                <mat-option *ngFor="let facility of filteredFacilities | async" [value]="facility">
-                  {{facility}}
-                </mat-option>
-              </mat-autocomplete>
-            </mat-form-field>
+          <div class="input-style">
+            <perun-web-apps-facility-search-select
+              (facilitySelected)="filterResources($event.name); "
+              [disableAutoSelect]="true"
+              [facilities]="facilities"
+              [selectPlaceholder]="'DIALOGS.ADD_MEMBER_TO_RESOURCE.FILTER'">
+            </perun-web-apps-facility-search-select>
           </div>
-          <div>
-            <mat-form-field class="input-style">
-              <mat-select
-                disableOptionCentering="true"
-                placeholder="{{ 'DIALOGS.ADD_MEMBER_TO_RESOURCE.SELECTED' | translate}}"
-                required>
-                <mat-option
-                  (click)="setResource(resource);
-                           stepper.selected.completed = true"
-                  *ngFor="let resource of filteredResources | async"
-                  [value]="resource">
-                  {{resource.name}}
-                </mat-option>
-              </mat-select>
-            </mat-form-field>
+          <div class="input-style">
+            <perun-web-apps-resource-search-select
+              (resourceSelected)="setResource($event);
+                         stepper.selected.completed = true"
+              *ngIf="!processing"
```

Comment:
```suggestion
```
think this is unnessacary (or is this meant to reset the search value on select? if so, could this be done without the search-selects disappearing (this seems quite ugly to me)? (I'm assuming this would require changes to the search-select components)